spec/serializable/serializable_submission_file_spec.rb

```diff
@@ -2,11 +2,16 @@
 
 RSpec.describe SerializableSubmissionFile do
   context 'given a submission file with an attachment' do
-    let(:submission_file) { FactoryBot.create(:submission_file, :with_attachment) }
+    let(:submission_file) { FactoryBot.create(:submission_file, :with_attachment, filename: 'not-really-an.xls') }
     let(:serialized_submission_file) { SerializableSubmissionFile.new(object: submission_file) }
 
     it 'exposes the filename of the attached file' do
       expect(serialized_submission_file.as_jsonapi[:attributes][:filename]).to eql 'not-really-an.xls'
     end
+
+    it 'exposes the temporary download URL of the attached file' do
+      expect(serialized_submission_file.as_jsonapi[:attributes][:temporary_download_url])
+        .to match(/not-really-an.xls$/)
```

Comment:
same as above, right now the test would pass if the method just returned the file name. Could we make it a bit more specific?